.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -11,5 +11,16 @@ assignees: ''
 Troubleshooting guide: https://rust-analyzer.github.io/manual.html#troubleshooting
 Forum for questions: https://users.rust-lang.org/c/ide/14
 
-Please try to provide information which will help us to fix the issue faster. Minimal reproducible examples with few dependencies are especially lovely <3
+Before submitting, please make that you're not running into one of these known issues:
+
+ 1. local imports (`use` statements) don't work: #1165
+ 2. local items don't work: #1559
+ 3. the `rust-lang.rust` (official extension) conflicts with `rust-analyzer`
+ 4. some platform-specific imports are not resolved: #6038
+ 5. some settings are required for procedural macro and build script support (`rust-analyzer.cargo.loadOutDirsFromCheck`, `rust-analyzer.procMacro.enable`): #6448
+ 6. on-the-fly diagnostics are mostly unimplemented (`cargo check` diagnostics will be shown when saving a file)
+ 7. some types are not resolved in trait-heavy code
```

Comment:
Is there a linked issue for this?

Reply:
Probably more than one, added a random one.